luminoso_api/constants.py

```diff
@@ -1,4 +1,4 @@
 from __future__ import unicode_literals
-API_VERSION=4
-API_HOST='api.luminoso.com'
-URL_BASE='https://%s/v%d' % (API_HOST, API_VERSION)
+API_VERSION = 4
```

Comment:
Is this constant actually used anywhere on its own?  Else I'd really suggest just folding it into `URL_BASE`.

It is also somewhat contradictory that we have these constants here but don't use them in `get_root_url()`.  Maybe this should actually just export a hardcoded full `URL_BASE`?